daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,11 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	exec.Command("modprobe", "bridge", "nf_nat").Run()
```

Comment:
maybe since this is such a weird issue that we are fixing, it might be a good idea to log the error ( along with stderr ) for this command with `logrus.Error` incase something else get weird we can see that this call failed.